docs/docs/en/guide/task/switch.md

```diff
@@ -14,26 +16,42 @@ Drag from the toolbar <img src="../../../../img/switch.png" width="20"/>  task n
 - Descriptive information: Describe the function of the node.
 - Task priority: When the number of worker threads is insufficient, execute in the order of priority from high to low, and tasks with the same priority will execute in a first-in first-out order.
 - Worker grouping: Assign tasks to the machines of the worker group to execute. If `Default` is selected, randomly select a worker machine for execution.
+- Environment name: The environment in Security, if not configured, it will not be used.
+- Task group name: The group in Resources, if not configured, it will not be used.
 - Times of failed retry attempts: The number of times the task failed to resubmit. You can select from drop-down or fill-in a number.
 - Failed retry interval: The time interval for resubmitting the task after a failed task. You can select from drop-down or fill-in a number.
+- Delay execution time: Task delay execution time.
 - Timeout alarm: Check the timeout alarm and timeout failure. When the task runs exceed the "timeout", an alarm email will send and the task execution will fail.
 - Condition: You can configure multiple conditions for the switch task. When the conditions are satisfied, execute the configured branch. You can configure multiple different conditions to satisfy different businesses.
 - Branch flow: The default branch flow, when all the conditions are not satisfied, execute this branch flow.
 
-## Detail
+## Task Example
 
-Here we have three tasks, the dependencies are `A -> B -> [C, D]`, and `task_a` is a shell task and `task_b` is a switch task
+This is demonstrated using one switch task and three shell tasks.
 
-- In task A, a global variable named `id` is defined through [global variable](../parameter/global.md), and the declaration method is `${setValue(id=1)}`
-- Task B adds conditions and uses global variables declared upstream to achieve conditional judgment (Note: switch can get the global variables value, as long as its direct or indirect upstream have already assigned the global variables before switch acquires). We want to execute task C when `id = 1`, otherwise run task D
-  - Configure task C to run when the global variable `id=1`. Then edit `${id} == 1` in the condition of task B, and select `C` as branch flow
-  - For other tasks, select `D` as branch flow
+![switch_01](/img/tasks/demo/switch_01.png)
 
-The following shows the switch task configuration:
+### Create a workflow
 
-![task-switch-configure](../../../../img/switch_configure.jpg)
+Create a new switch task, and three shell tasks downstream. The shell task is not required.
+The switch task needs to be connected with the downstream task to configure the relationship before the downstream task can be selected.
 
-## Related Task
+![switch_01](/img/tasks/demo/switch_01.png)
```

Comment:
You should use related path instead of absolute path after #10325 merged

Reply:
I had change the img path.